src/main/resources/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig/help-usePathStyleUrl.html

```diff
@@ -0,0 +1,6 @@
+<div>This option modifies the format of the URL used to communicate with S3.<br />
+<br />
+When this option is disabled URLs will be formatted https://bucketname.s3.amazonaws.com/key/path<br />
+When this option is enabled URLs will be formatted https://s3.amazonaws.com/bucketname/key/path<br />
+Typically, you would want this enabled when you're using the real S3 but disabled when using an S3 compatible service.
```

Comment:
Wait, what? According to [documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html) everyone should use virtual host style on (actual) S3, i.e., this option should be disabled. IIUC you would _enable_ it (thus setting `PROPERTY_S3_VIRTUAL_HOST_BUCKETS` to `false`) in case you were using a provider which only supported path-style URLs, or you had difficulty making a DNS wildcard work, as suggested [here](https://jclouds.apache.org/guides/aws-s3/). So is this line just backwards?

Reply:
Yes it is, I will update it

Reply:
Done, I hope it is more clearer now